src/sdk.ts

```diff
@@ -47,11 +62,12 @@ class KontentSmartLinkSDK {
 
     this.domSmartLinkManager = new DOMSmartLinkManager(this.iframeCommunicator, this.configuration);
 
-    this.initialize();
+    // biome-ignore lint/complexity/noVoid: intentionally used void
```

Comment:
I would be in favor of removing the `noVoid` from the biome config. We are clever people and know how this operator works.